float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax
int64 area
int16 id
string Class